static/js/settings.js

```diff
@@ -164,6 +164,22 @@ function _setup_page() {
         });
     }
 
+    var old_statuses = {};
+    if (keep_status) {
```

Comment:
Since no where this is invoked with `keep_status` it look like the following lines won't be executed. Or am I missing something?